common/templates/options.php

```diff
@@ -2,6 +2,8 @@
 
 use Valued\WordPress\WooCommerce;
 
+$plugin->setDefaultInviteOption();
```

Comment:
I don't think there's any point in this, since at the point that this is called, `$config` is already set, so it will only impact it on the next load.

Reply:
This is intended to be edge case prevention. The idea here is that when the config page is opened, the radio button option is already chosen, but not saved in the db. When users open it, they see it's already chosen and may not save the config, which leads to invites not being sent.

Reply:
In principle that should never happen since they would then also not have configured their API key, but I see what you mean.

But now, since we have added AFTER_EVERY_ORDER as a default now in getDefaultConfig, that should no longer be possible, and this code can be removed. Right?